concepticondata/conceptlists/Anonby-2018-1500.tsv

```diff
@@ -0,0 +1,1501 @@
+Anonby-2018-1500-1138	1138	658	RAIN (PRECIPITATION)	rain	باران	bārān	Environment	Weather	
+Anonby-2018-1500-1139	1139	1016	DRIZZLE	drizzle	نم نمِ باران	namnam-e bārān	Environment	Weather	
+Anonby-2018-1500-1140	1140	609	HAIL	hail	تگرگ	tagarg	Environment	Weather	
+Anonby-2018-1500-1141	1141	1977	DEW	dew	شبنم	sabnam	Environment	Weather	
+Anonby-2018-1500-1142	1142	613	HIGH TIDE	flood (n.)	سیل	sayl	Environment	Weather	
+Anonby-2018-1500-1143	1143	1786	DRY UP	dry up, evaporate	تبخیر شدن	tabxir šodan	Environment	Weather	
+Anonby-2018-1500-1144	1144	199	FAMINE	drought, famine	خشکسالی	xošksāli	Environment	Weather	
+Anonby-2018-1500-1145	1145	691	SEASON	season	فصل	fasl	Environment	Weather	Seasons
+Anonby-2018-1500-1146	1146	1537	SPRINGTIME	spring	بهار	bahār	Environment	Weather	Seasons
+Anonby-2018-1500-1147	1147	1342	SUMMER	summer	تابستان	tābestān	Environment	Weather	Seasons
+Anonby-2018-1500-1148	1148	1276	AUTUMN	autumn, fall	پاییز	pāyiz	Environment	Weather	Seasons
+Anonby-2018-1500-1149	1149	1471	WINTER	winter	زمستان	zemestān	Environment	Weather	Seasons
+Anonby-2018-1500-1150	1150	630	LIGHT (RADIATION)	light	نور	nur	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1151	1151	3276	SUNSHINE	sunshine	نورِ خورشید	nur-e xoršid	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1152	1152			moonlight	نورِ ماه	nur-e māh	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1153	1153	1388	SHADE	shadow	سایه	sāye	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1154	1154	11	DARKNESS	darkness	تاریکی	tāriki	Environment	Weather	Ambient conditions
+Anonby-2018-1500-1155	1155	892	TIME	time	وقت، زمان	ʋaġt, zamān	Environment	Time	
+Anonby-2018-1500-1156	1156	1376	NOW	now	حالا، اکنون	hālā, aknun	Environment	Time	
+Anonby-2018-1500-1157	1157	1882	BEFORE	before	قبل، پیش از	ġabl, piš az	Environment	Time	
+Anonby-2018-1500-1158	1158	1685	AFTER	after	بعد، پس از	ba’d, pas az	Environment	Time	
+Anonby-2018-1500-1159	1159	672	EARLY	early	زود	zud	Environment	Time	
+Anonby-2018-1500-1160	1160	477	LATE	late	دیر	dir	Environment	Time	
+Anonby-2018-1500-1161	1161	374	ONE TIME	once	یک بار، یک باره 	yekbār, yekbāre	Environment	Time	
+Anonby-2018-1500-1162	1162	204	AGAIN	again	دوباره، باز	dobāre, bāz	Environment	Time	
+Anonby-2018-1500-1163	1163	1544	SOMETIMES	sometimes	بعضی اوقات	ba’zi aʋġāt	Environment	Time	
+Anonby-2018-1500-1164	1164	1852	OFTEN	often	اغلب	aġlab	Environment	Time	
+Anonby-2018-1500-1165	1165			usually	معمولاً	ma’mulan	Environment	Time	
+Anonby-2018-1500-1166	1166	1676	ALWAYS	always	همیشه	hamiše	Environment	Time	
+Anonby-2018-1500-1167	1167	1656	NEVER	never	هرگز، هیچ وقت	hargez, hičʋaġt 	Environment	Time	
+Anonby-2018-1500-1168	1168			spend time, pass time	وقت گذراندن	ʋaġt gozarāndan	Environment	Time	
+Anonby-2018-1500-1169	1169	1590	WAIT (FOR)	wait	انتظار کشیدن، منتظر بودن	entezār kešidan, montazer budan	Environment	Time	
+Anonby-2018-1500-1170	1170	1225	DAY (NOT NIGHT)	day	روز	ruz	Environment	Time	Time periods
```

Comment:
Maybe this one is DAY (24 HOURS) 1260 while Anonby-2018-1500-1186 is DAY (NOT NIGHT) also because the latter is followed directly by 'night'?

Reply:
Yes that would make sense, also because the next entries after this one are month and year.